docs/usermanual/userman_admin.rst

```diff
@@ -565,7 +565,7 @@ Restarting Your Server: Linux
 
 #. Navigate to the folder that you downloaded KA Lite in. You can do this by typing ``cd <PATH/TO/KALITE/FOLDER>``. Change the path to the path name of the KA Lite folder on your machine.
```

Comment:
Don't need this step, since we're using the "kalite" command that's on the PATH.